chainerx_cc/chainerx/python/array.cc

```diff
@@ -85,6 +86,28 @@ py::array MakeNumpyArrayFromArray(const ArrayBodyPtr& self, bool copy) {
     return py::array{dtype, shape, strides, ptr, py::cast(internal::MoveArrayBody(std::move(array)))};
 }
 
+py::object MakeCupyArrayFromArray(const py::module& m, py::handle self) {
+    Array array = Array{py::cast<ArrayBodyPtr>(self)};
+    const Device& device = array.device();
```

Comment:
`Device&` is never const.
```suggestion
    Device& device = array.device();
```